troposphere/ec2.py

```diff
@@ -757,13 +757,47 @@ class LaunchSpecifications(AWSProperty):
     }
 
 
+class SpotFleetLaunchTemplateSpecification(AWSProperty):
+    props = {
+        'LaunchTemplateId': (basestring, False),
+        'LaunchTemplateName': (basestring, False),
+        'Version': (basestring, True),
```

Comment:
Docs say not required, but CF gives an error when not specified, and Version is required in both Instances and ASGs when referencing a Launch Template. See https://github.com/awsdocs/aws-cloudformation-user-guide/pull/162